types.d.ts

```diff
@@ -10643,7 +10652,7 @@ declare abstract class SortableSet<T> extends Set<T> {
 declare class Source {
 	constructor();
 	size(): number;
-	map(options?: MapOptions): Object;
+	map(options?: MapOptions): null | RawSourceMap;
```

Comment:
`null` can be returned here https://github.com/webpack/webpack-sources/blob/main/lib/RawSource.js#L43